pipeline/variables.sample.yml

```diff
@@ -1,61 +1,114 @@
+  
 #
 # Terraform
 #
 
 # Amazon access to use to run terraform. Usually admin account to be able to create Amazon services.
+
+#. aws_access_key (required): ((aws.access_key))
+#+ Amazon AWS access key for Terraform. see value format [Here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)
 aws_access_key: ((aws.access_key))
+
+#. aws_secret_key (required): ((aws.secret_key))
+#+ Amazon AWS secret key for Terraform. see value format [Here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)
 aws_secret_key: ((aws.secret_key))
-aws_default_region: eu-west-1
 
-# Password used for your rds. Set "empty" if you dont use databases
-rds_password: ((raw_rds_password))
+#. aws_default_region (required): eu-west-1
+#+ Amazon AWS region to use for Terraform.
+aws_default_region: eu-west-1
 
-# Bucket configuration to store terraform remote state file.
+#. terraform_storage_bucket_name (required): ($ organization_canonical $)-terraform-remote-state
+#+ AWS S3 bucket name to store terraform remote state file.
 terraform_storage_bucket_name: ($ organization_canonical $)-terraform-remote-state
+
+#. terraform_storage_bucket_path (required): ($ project $)/($ environment $)
+#+ AWS S3 bucket path to store terraform remote state file.
 terraform_storage_bucket_path: ($ project $)/($ environment $)
 
+#. rds_password (optional): ((raw_rds_password))
+#+ Password used for your rds. Set "empty" if you dont use databases
+rds_password: ((raw_rds_password))
+
+
 #
 # Ansible
 #
 
+#. deploy_bucket_name (required): ($ project $)-deploy
+#+ AWS S3 bucket name in which store the builded code of the website.
 deploy_bucket_name: ($ project $)-deploy
+
+#. deploy_bucket_object_path (required): /catalog-lemp-app/($ environment $)/lemp-app.tar.gz
+#+ AWS S3 bucket path in which store the builded code of the website.
 deploy_bucket_object_path: /catalog-lemp-app/($ environment $)/lemp-app.tar.gz
 
-# Password used for your ansible vault variables (raw_ansible_vault_password depend of you credential name).
+#. ansible_vault_password (required): ((raw_ansible_vault_password))
+#+ Password used by ansible vault to decrypt your vaulted files.
 ansible_vault_password: ((raw_ansible_vault_password))
 
-# Configuration of the bastion server used to run ansible (ssh_bastion depend of you credential name)
+#. bastion_url (required): admin@bastion.cycloid.io
+#+ bastion URL used by ansible to connect on AWS EC2 instances.
```

Comment:
Shall we set this to such default value? As it's required might not make sense?

Reply:
ok let's put something better as example